core/src/main/java/hudson/model/TimeZoneProperty.java

```diff
@@ -0,0 +1,118 @@
+package hudson.model;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import hudson.util.ListBoxModel;
+import hudson.util.ListBoxModel.Option;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.TimeZone;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+
+/**
+ * A UserProperty that allows a user to specify a time zone for displaying time.
+ *
+ * @author Nathan Grubb
```

Comment:
Remove

Reply:
What exactly do you want removed? @daniel-beck

Reply:
Remove the `@author` please. We do not generally use them anywhere. This is generally misleading, and git blame is better if we need to ask questions.